### DISCUSSION TITLE
How to use Pythonw on MacOS?

### DISCUSSION THREAD

**Discussion by JacobBumgarner** [pyenv/pyenv#2291] [Q&A]
When trying to run a program with Pythonw on MacOS, the default /usr/bin/pythonw which links to `pythonw2.7` is called. 

Is there some way to run pythonw from the activated virtualenv?

**Comment by native-api** [comment]
We create shims for whatever executables Python's build produces.
AFAICS in https://github.com/python/cpython/blob/main/Mac/README.rst, a separate `pythonw` is no longer needed so they stopped producing it some time ago.

**Reply by JacobBumgarner** [reply]
Hmm - ok. I was able to run the GUI application that I'm working with after reinstalling python with the --enable-framework flag. Specifically, I'm working with [DeepLabCut](https://github.com/DeepLabCut/DeepLabCut/issues/1742).

There is a crash occurring when using the package's GUI, but I'm not sure whether it's a package issue or a pyenv issue. The crash that I'm running into doesn't occur when using an Anaconda environment. This is leading me to believe that the issue may have something to do with pyenv, but frankly I don't know enough about environment management to say that confidently.

**Reply by native-api** [reply]
FWIW our issue template has a list of the necessary diagnostic data that we need to diagnose a specific issue with Pyenv.

**Reply by JacobBumgarner** [reply]
I understand. I hesitated to create an issue for this topic, because I wasn't even sure if it actually was a Pyenv issue. Hopefully that makes sense! If you don't think that I'd be spamming the issues by opening a new ticket for this, I can go ahead and do that.
